Software/Python/README.md

```diff
@@ -1,35 +1,25 @@
-GrovePi
-=======
+## Installing the GrovePi for Python
 
-GrovePi is an open source platform for connecting Grove Sensors to the Raspberry Pi.
+Run the following 2 commands in the following order to get the GrovePi installed:
+```
+pip install -r requirements.txt
+```
+```
+python3 setup.py install
+```
 
-See more at the [GrovePi Site](http://dexterindustries.com/GrovePi/)
-[Dexter Industries](http://www.dexterindustries.com)
+You can also run `python setup.py test` to test import the modules of the GrovePi package that are listed in the [package_modules.txt](package_modules.txt) file. The `python setup.py test` commands should be run after pip installing the dependencies.
 
-## License
+## Library Breakdown
```

Comment:
is the  MIT license removed?

Reply:
I know that the MIT license is only needed to be added in one place, the root directory, like here:
https://github.com/DexterInd/GrovePi/blob/master/LICENSE

It would be inefficient to have to add it everywhere, right?

Reply:
I think we can drop a one-liner in each file. Those files are often copied and moved out of the root directory.

Reply:
One liner like what? What do you have in mind? Like one saying "This module is part of the GrovePi package"?

Reply:
# Released under the MIT license (http://choosealicense.com/licenses/mit/).
# For more information see https://github.com/DexterInd/GoPiGo3/blob/master/LICENSE.md

Reply:
ok, so a 2-liner

Reply:
Yeah I agree with that. Can't hurt at all and we don't get that wall text.